### ISSUE TITLE
Issue causing Server Error 500 should not block from navigating the site just like an unauthenticated session would  

### ISSUE THREAD

**Issue by sgornick** [stackernews/stacker.news#114]
When attempting to register with a new account on SN today, clicking the Sign-in button from the e-mail resulted in a Server Error 500 page.

But then the site was unusable.  Clicking the Login button, or any of the links from the header section will flash the message:
    "Application error: a client-side exception has occurred (see the browser console for more information).
and then right back to the Server Error page.

In the Javascript console, there is an HTTP 500 response to each request, so I can see why this Server Error keeps appearing.

But why wouldn't the home page, or login button, respond just like I'm unauthenticated if it is having an issue with my authentication?

So this issue is essentially reporting how once there's an issue with authentication, the only way to get back to the home page or a login page is to clear cookies (presumably that will work, .... I simply close and restart my browser in incognito mode.)   

So to summarize, when the Server Error 500 is not caused by a site-wide issue, but something specific to that session, there should be some way to continue to navigate the site, or some instruction such as "if this error recurs, clear cookies" or something like that.

**Comment by huumn** [comment]
Agreed! Good issue, thanks

**Comment by huumn** [comment]
We haven't encounter this issue for awhile so closing.
